arc/src/arc_curie_pme.c

```diff
@@ -51,7 +53,8 @@ u16_t curie_pme_learn(u8_t *pattern_vector,
 
     // Mask off the 15th bit-- valid categories range from 1-32766,
     // and bit 15 is used to indicate if a firing neuron has degenerated
-    reg_write16(CAT, (reg_read16(CAT) & ~CAT_CATEGORY) | (category & CAT_CATEGORY));
+    reg_write16(CAT,
+                (reg_read16(CAT) & ~CAT_CATEGORY) | (category & CAT_CATEGORY));
```

Comment:
again, this is automatically done by clang, I couldn't find a rule to do this automatically for me, but I guess I can do it manually.